include/gzip/utils.hpp

```diff
@@ -1,4 +1,7 @@
+#include <cstdlib>
+
```

Comment:
This include is needed for this header to compile on its own, since `uint8_t` is used.